src/MBCs.md

```diff
@@ -20,3 +20,9 @@ so, it might be nevertheless possible to use Double Speed during periods
 which use only code and data which is located in internal RAM. Despite the 
 above, a self-made MBC1-EPROM card appears to work stable and fine even in 
 Double Speed Mode.
+
+## MBC Out Of Bounds RAM Access
+
+On most of the MBC's in case an out of bounds address is accessed (selecting unmapped RAM bank),
+only the mapped bits from the address will be used.
```

Comment:
Wait--is this talking about address, or bank? I know they are combined to ultimately make a single address passed to the chip; but the entire document is written from the perspective of the GB CPU, since that's the most consistent and generally-useful one, and from that perspective, the bank and address are separate.

Reply:
Agree! Just rephrased it to be more CPU oriented, let me know what you think